backend/apps/owasp/graphql/nodes/project.py

```
@@ -72,3 +80,13 @@ def resolve_repositories_count(self, info):
     def resolve_topics(self, info):
         """Resolve topics."""
         return self.idx_topics
+
+    def resolve_top_contributors(self, info, limit=15, excludedUsernames=None):
```

Ruff (0.8.2)

Argument name `excludedUsernames` should be lowercase

(N803)